src/main/plugins/org.dita.base/xsl/common/strings-sl-si.xml

```diff
@@ -33,6 +33,7 @@ See the accompanying LICENSE file for applicable license.
    <variable id="Value">Vrednost</variable>
    <variable id="Note">Opomba</variable>
    <variable id="Notes">Opombe</variable>
+   <variable id="Notice">Notice</variable><!--TODO:Notice-->
```

Comment:
In the original `sl.xml` this one was
`<variable id="Notice">Objava</variable>`